syntax = "proto2";
package common;

message RowDefinition {
  required string object_id = 1;
  map<string, string> fields = 2;
}
